src/yoroi-wallets/types/types.ts

```diff
@@ -57,3 +57,19 @@ export type YoroiAmount = {
 export type YoroiMetadata = {
   [label: string]: string
 }
+
+export type YoroiNFT = {
```

Comment:
is this type defined in a library shared by mobile and extension?

Reply:
I didn't found it, maybe we should include it in the `yoroi-lib` package